Subject: [PATCH] m68k: FB_HP300 depends on DIO

From: Geert Uytterhoeven <geert@linux-m68k.org>

FB_HP300 doesn't depend only on HP300, but also on DIO (which depends on HP300)

Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
---
 drivers/video/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/video/Kconfig
+++ b/drivers/video/Kconfig
@@ -596,7 +596,7 @@ config FB_MAC
 
 config FB_HP300
 	bool
-	depends on (FB = y) && HP300
+	depends on (FB = y) && DIO
 	select FB_CFB_IMAGEBLIT
 	default y
 
